debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCObjectHeapIteratorAddressOrderedList_V1.java

```diff
@@ -137,7 +131,20 @@ public int compare(U8Pointer[] o1, U8Pointer[] o2)
 		excludedRangeList.toArray(excludedRanges);
 		currentExcludedRange = 0;
 	}
-	
+
+	protected void addExcludedRangeForDisablingTLHOrSamplingCase(boolean nonZeroTLH, J9VMThreadPointer vmThread, U8Pointer base, U8Pointer top, U8Pointer heapAlloc, U8Pointer heapTop, ArrayList<U8Pointer[]> excludedRangeList)
+	{
+		U8Pointer realHeapAlloc = null;
+		if (nonZeroTLH) {
+			realHeapAlloc = adjustedToRange(vmThread.nonZeroAllocateThreadLocalHeap().realHeapAlloc(), base, top);
+		} else {
+			realHeapAlloc = adjustedToRange(vmThread.allocateThreadLocalHeap().realHeapAlloc(), base, top);
```

Comment:
This won't compile: there's no (generated) method named `realHeapAlloc` in current builds.